beets/ui/commands.py

```diff
@@ -708,6 +729,14 @@ def choose_match(self, task):
                     _, _, candidates, rec = autotag.tag_album(
                         task.items, search_id=search_id
                     )
+            elif choice in extra_ops.keys():
+                # Allow extra ops to automatically set the post-choice.
+                post_choice = extra_ops[choice](self, task)
+                if post_choice in (importer.action.SKIP,
+                                   importer.action.ASIS,
+                                   importer.action.TRACKS,
+                                   importer.action.ALBUMS):
```

Comment:
Maybe `isinstance(post_choice, importer.action)` would be fine—as long as we kindly ask people not to use `MANUAL` and `MANUAL_ID`?